.github/workflows/link-smoke-tests.yaml

```diff
@@ -0,0 +1,99 @@
+name: Lint and Smoke Tests
+
+on:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  push:
+    branches:
+      - master
+
+env:
+  TIMEOUT: 15m
+  TYK_OSS_NS: tyk-oss
+  TYK_DATAPLANE_NS: tyk-dataplane
+  TYK_STACK_NS: tyk-stack
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+        with:
+          version: v3.3.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml --all
+
+  smoke-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: ["v1.26.13", "v1.27.10","v1.28.6","v1.29.1"]
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+
+      - name: Create Kind Cluster
+        uses: helm/kind-action@v1.4.0
+        with:
+          node_image: "kindest/node:${{ matrix.k8s_version }}"
+
+      - name: Install helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.2
+
+      - name: Deploy Tyk OSS and Dependencies
+        run: |
+          kubectl create namespace tyk-oss
+
+          # Do not change the name
+          helm install redis simple-redis --repo 'https://helm.tyk.io/public/helm/charts/' -n tyk-oss --timeout ${{ env.TIMEOUT }}
```

Comment:
will we continue using simple-redis in smoke tests?

Reply:
Will fix that